sidebar.less

```diff
@@ -183,22 +183,22 @@
 	// Subscribe buttons (RES wraps in .subButtons)
 	.subscribe-button {
 		order: 1;
-		
+		margin-right: 0px;
```

Comment:
This shouldn't be cleared to 0. The inherited value is 5px, which you set again below.